front/src/common/Pathfinding/pathfinding.scss

```diff
@@ -0,0 +1,70 @@
+.pathfinding-main-container {
+  margin-left: 1.5rem;
+
+  .pathfinding-subsection {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .pathfinding-done {
+    @extend .pathfinding-subsection;
+    
+    .title {
+      color: green;
```

Comment:
please use Bootstrap's color with var(--green) for example

Reply:
done